x-pack/plugins/security/public/components/management/users/edit_user.js

```diff
@@ -289,6 +311,11 @@ export class EditUser extends Component {
     if (!user || !roles) {
       return null;
     }
+
+    if (!this.state.isLoaded) {
+      return null;
```

Comment:
Seems like this should return some sort of loading indicator instead of a blank page.